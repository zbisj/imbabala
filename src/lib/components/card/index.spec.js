import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Card from ".";

describe("Card", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <Card
          image="https=//bit.ly/3Au6jS4"
          heading="Heading"
          content="Irure aliquip qui cupidatat veniam aliquip id deserunt id irure Lorem do minim eiusmod. Mollit irure incididunt anim fugiat cillum deserunt ex commodo enim pariatur. Dolor ut ea aliquip pariatur dolore esse laboris enim laborum do et velit."
          actions={[{ label: "Click Me", variant: "primary" }]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
