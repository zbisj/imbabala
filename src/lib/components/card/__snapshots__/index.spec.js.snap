// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className=""
>
  <img
    className=""
    src="https=//bit.ly/3Au6jS4"
  />
  <div
    className=""
  >
    <h4
      className=""
    >
      Heading
    </h4>
    <p>
      Irure aliquip qui cupidatat veniam aliquip id deserunt id irure Lorem do minim eiusmod. Mollit irure incididunt anim fugiat cillum deserunt ex commodo enim pariatur. Dolor ut ea aliquip pariatur dolore esse laboris enim laborum do et velit.
    </p>
    <div
      className=""
    >
      <button
        className=""
      >
        Click Me
      </button>
    </div>
  </div>
</div>
`;
